pkg/metadata/host/host.go

```diff
@@ -272,7 +272,7 @@ func getProxyMeta() *ProxyMeta {
 
 	return &ProxyMeta{
 		NoProxyNonexactMatch: config.Datadog.GetBool("no_proxy_nonexact_match"),
```

Comment:
Could be done in a different PR, but: in addition to the config setting's value, we should also send whether the config setting is explicitly set or not in the Agent config, because this will determine whether changing the default value of the config setting in a future agent version will be a breaking change for users.

In order to do this, you have to _not_ set a default value for this config option in `config/config.go`, and then `IsSet` will return whether the option is explicitly configured or not.